src/Maestro/Maestro.Web/Startup.cs

```diff
@@ -435,6 +436,14 @@ public override void Configure(IApplicationBuilder app)
         }
         else
         {
+            app.UseWhen(context => context.Request.Path == AccountController.AccountSignInRoute, app =>
```

Comment:
@dkurepa one last thing - put a comment here explaining why we're doing this

Reply:
added

Reply:
@dkurepa this explains what the code does and what is easily visible from the code. The comments should instead explain what is not clear from the code - so the context that is about GitHub redirecting back to HTTP with the token inside which is not safe.

Reply:
Also it should say "authenticating"